### PR TITLE
[9.0] extract project_department

### PR DESCRIPTION
**Pull request by yvaucher**
Moving https://github.com/OCA/department/pull/53 here

Depends on
- [x] OCA/account-analytic#57

### CODE REVIEW

File: project_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Project Department Categorization",
+    "version": "9.0.1.0.0",
+    "author": "Camptocamp, Daniel Reis,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Generic Modules/Projects & Services",
+    "description": """\
+Add Department to Projects and task to corresponding tree, search and form
+views.
```
Comment (yvaucher) [inline project_department/__openerp__.py]: to move in README.rst

Comment (dreispt) [inline project_department/__openerp__.py]: +1

**Comment by rafaelbn** [comment]
Hi, @yvaucher just please review README and travis, dependency is merged. We can test it! Thanks!

**Comment by thinkwelltwd** [comment]
I would like to migrate this module to v10, since our server is not on v9. Is there any reason v9 can't be merged? It looks like the tests are passing.

**Comment by yvaucher** [comment]
@thinkwelltwd There is just the description to move in `README.rst`

I will fix that in order to let you do the migration for 10.0

**Comment by thinkwelltwd** [comment]
> I will fix that in order to let you do the migration for 10.0

Ok great, should I wait till this is merged before starting?

**Comment by yvaucher** [comment]
@thinkwelltwd Just did the change you can rely on those last commits to make your migration

**Comment by dreispt** [comment]
@yvaucher Can you fix the README?

File: project_department/LICENSE [hunk 2]
```diff
@@ -0,0 +1,18 @@
+OpenERP, Open Source Management Solution
```
Comment (elicoidal) [inline project_department/LICENSE:1]: No need this file AFAIK

File: project_department/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Daniel Reis
```
Comment (elicoidal) [inline project_department/README.rst:41]: Add email @dreispt

**Review by elicoidal — CHANGES_REQUESTED** [review]
Some details

**Comment by yvaucher** [comment]
@dreispt can you be more specific? what should I fix in the readme?

**Review by dreispt — APPROVED** [review]

**Comment by dreispt** [comment]
Sorry @elicoidal your concerns are minor, so I prefer to move on and merge.

**Comment by elicoidal** [comment]
:+1:

**Pull request merged**